cpp/src/barretenberg/honk/sumcheck/sumcheck_output.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <array>
+#include <vector>
+namespace honk::sumcheck {
+
+/**
+ * @brief Contains the multi-linear `evaluations` of the polynomials at the `evaluation_point`.
+ * These are computed by the prover and need to be checked using a multi-linear PCS like Gemini.
+ */
+template <typename FF> struct SumcheckOutput {
+    // u = (u_0, ..., u_{d-1})
+    std::vector<FF> multivariate_query;
```

Comment:
Yeah can we call this something else?

Reply:
It's a challenge point.

Reply:
Ok, so if we have v = f(z) (for either univariate or multilinear) we would have z = `challenge` and v = `evaluation`?